src/Helpers/Constants.cs

```diff
@@ -0,0 +1,12 @@
+using System.Globalization;
+
+public class Constants
+{
+    public static readonly long MINIMUM_CHANNEL_CAPACITY_SATS;
+    
+    static Constants()
+    {
+        var environmentVariable = Environment.GetEnvironmentVariable("MINIMUM_CHANNEL_CAPACITY_SATS") ?? throw new InvalidOperationException("MINIMUM_CHANNEL_CAPACITY_SATS not set");
+        MINIMUM_CHANNEL_CAPACITY_SATS = long.Parse(environmentVariable, NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture);
```

Comment:
You should use tryparse, parse is dangerous and can throw exceptions

Reply:
The idea is that it throws an exception, though. It would be nice to have this class throw before everything else is executed so you know that you forgot an env var right away

Reply:
Fair enough